sky/utils/kubernetes/rsync_helper.sh

```diff
@@ -1,7 +1,16 @@
-# When using pod@namespace, rsync passes args as: {us} -l pod namespace 
+# When using pod@namespace/context, rsync passes args as: {us} -l pod namespace+context
```

Comment:
Seems we are using `namespace+context` when we are calling rsync. Should we update this comment?
```suggestion
# When using pod@namespace+context, rsync passes args as: {us} -l pod namespace+context
```